### PR TITLE
UserProfile activity on user image click in header

### LINKED ISSUE #67: feat: clicking on person avatar opens customer profile activity
**Summary:**
The User should be able to view their profile picture on clicking on the picture icon in Dashboard Fragment

**Steps to reproduce:**
Go to dashboard activity click on the person avatar
**Expected behaviour:**
Customer profile activity opens

**Observed behaviour:**

Nothing happens
**Logcat for the issue**

Provide logs for the crash here

**Device and Android version:**

Oxygen OS 9.0.17  for OnePlus6t Mclaren
**Screenshots:**

ezgif com-video-to-gif

### PR DESCRIPTION
**Pull request by PiyushSharma99**
Fixes #67 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt [hunk 1]
```diff
@@ -178,6 +187,9 @@ class DashboardActivity : MifosBaseActivity(), View.OnClickListener, NavigationV
             R.id.item_about_us -> {
                 showAboutUs()
             }
+            R.id.item_settings -> {
+                replaceFragment(SettingsFragment.newInstance(), true, R.id.container)
+            }
```
Comment (jddeep) [inline app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt]: @PiyushSharma99 please remove your previous change for the settings fragment from this PR.

Comment (PiyushSharma99) [reply app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt]: Done @jddeep Thanks!

**Comment by garvit984** [comment]
Duplicate #74. So we are closing it.

**Comment by garvit984** [comment]
@abhijitramesh This is a duplicate PR.

**Comment by abhijitramesh** [comment]
Closing this PR as it is a duplicate

**Pull request closed without merging**